templates/index.html

```diff
@@ -32,362 +30,475 @@
+                            alt="",
+                            width="900",
+                            height="506",
+                            hi_def=True,
+                            attrs={
+                            "class": "p-image-container__image",
+                            },
+                            loading="auto") | safe
+              }}
             </div>
           </div>
         </div>
       </div>
-    </section>
-    <section class="p-section">
-      <div class="row">
-        <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
-          <hr class="p-rule" />
-          <div class="row">
-            <div class="col-6">
-              <h3 class="p-heading--2">
-                Optimized
-                <br class="u-hide--medium u-hide--small" />
-                for IoT and edge
-              </h3>
-              <p>
-                Speed up time-to-market and lower maintenance costs. Ship devices that are self-healing, secure and
-                always up-to-date.
-              </p>
-            </div>
-            <div class="col-3">
-              <p class="p-heading--2">
-                <a href="https://ubuntu.com/core">Ubuntu Core&nbsp;&rsaquo;</a>
-              </p>
-              <hr class="p-rule--muted u-hide--medium" />
-              <p class="p-heading--2">
-                <a href="https://microk8s.io/">MicroK8s&nbsp;&rsaquo;</a>
-              </p>
-            </div>
-          </div>
+    </div>
+  </section>
+
+  <section class="p-section--deep">
+    <div class="row">
+      <div class="col-medium-6 col-9 col-start-large-4">
+        <div class="p-section--shallow">
+          <h2>Innovate on your own terms</h2>
+          <p>
+            Build your stack your way with composable open source solutions, all backed by the experts behind the world’s leading Linux. At the nexus of open source innovation and enterprise support, you can move fast without compromises.
+          </p>
         </div>
       </div>
-    </section>
-    <section class="p-section">
-      <div class="row">
-        <div class="col-start-medium-3 col-medium-4 col-start-large-4 col-9">
-          <hr class="p-rule" />
-          <div class="row">
-            <div class="col-6">
-              <h3 class="p-heading--2">Multi-cloud data and AI</h3>
-              <p>
-                Focus more on innovation and simplify operations with our multi-cloud data and AI solutions. Get the
-                predictable economics you need to scale your projects.
-              </p>
-            </div>
-            <div class="col-3">
-              <p class="p-heading--2">
-                <a href="/data">Data&nbsp;&rsaquo;</a>
-              </p>
-              <hr class="p-rule--muted u-hide--medium" />
-              <p class="p-heading--2">
-                <a href="/solutions/ai">AI and MLOps&nbsp;&rsaquo;</a>
-              </p>
+    </div>
+    <div class="p-section--shallow">
```

Comment:
This section still looks a bit off on med screens (tablet breakpoint on dev tools):

![image](https://github.com/user-attachments/assets/9af0fbe1-63d9-46ee-b2b7-645a95e2e697)

There's also what looks to be unintentional spacing between animations and non-animated elements on mobile:
![image](https://github.com/user-attachments/assets/dc8bfebd-8386-4962-993a-c4b1b50f2a88)